db/migrate/20241104053017_add_ai_artifacts.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+class AddAiArtifacts < ActiveRecord::Migration[7.1]
+  def change
+    create_table :ai_artifacts do |t|
+      t.integer :user_id, null: false
+      t.integer :post_id, null: false
+      t.string :name, null: false, limit: 255
+      t.string :html, limit: 65_535 # ~64KB limit
+      t.string :css, limit: 65_535 # ~64KB limit
+      t.string :js, limit: 65_535 # ~64KB limit
```

Comment:
Should these limits be enforced in the model with validators so it doesn't end up erroring in the database?

Reply:
will add limits, though it is probably 1 year before LLMs can fill this much :)